### PR TITLE
HMAN-368 temporary PR to test def file verion [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by mattnayler**
### JIRA link (if applicable) ###

   [HMAN-368](https://tools.hmcts.net/jira/browse/HMAN-368) _"Ccd Def store definition changes"_

### Change description ###
Temporary PR to test hmcts/ccd-test-definitions#53 def file changes and hmcts/befta-fw#190.

> NB: BEFTA FW & Def file FTA+ES test PRs are:
> * hmcts/ccd-data-store-api#2086
> * hmcts/ccd-definition-store-api#1279
> * hmcts/ccd-user-profile-api#497

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -19,7 +19,8 @@ def branchesToSync = ['demo', 'ithc', 'perftest']
 
 // Variables to switch pipeline logic and wiring per type of build
 
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
+// TODO: remove TEMPORARY OVERRIDE to test GlobalSearch, reinstate: def definitionStoreDevelopPr = "PR-575"
+def definitionStoreDevelopPr = "PR-1279"  // This doesn't change frequently, but when it does, only change this value.
```
Comment (mattnayler) [inline Jenkinsfile_CNP:23]: This should be reveresed if final test-def version changes are merged to `master`.

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -94,7 +95,8 @@ env.CASE_DOCUMENT_AM_URL = "http://ccd-case-document-am-api-aat.service.core-com
 env.RD_LOCATION_REF_API_BASE_URL = "http://rd-location-ref-api-aat.service.core-compute-aat.internal"
 env.RD_PROFESSIONAL_API_BASE_URL = "http://rd-professional-api-aat.service.core-compute-aat.internal"
 env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
-env.ELASTIC_SEARCH_FTA_ENABLED = "false"
+// TODO: remove TEMPORARY OVERRIDE to test GlobalSearch, reinstate: env.ELASTIC_SEARCH_FTA_ENABLED = "false"
+env.ELASTIC_SEARCH_FTA_ENABLED = "true"
```
Comment (mattnayler) [inline Jenkinsfile_CNP:99]: This should be reveresed if final test-def version changes are merged to `master`.

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 3]
```diff
@@ -27,7 +27,8 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.auth.password}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575-java/
+    # TODO: remove TEMPORARY OVERRIDE to test GlobalSearch, reinstate: DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575-java/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-1279-java/
```
Comment (mattnayler) [inline charts/ccd-data-store-api/values.preview.template.yaml:31]: This should be reveresed if final test-def version changes are merged to `master`.

File: Jenkinsfile_CNP (hunk 2 above)
Comment (mattnayler) [inline Jenkinsfile_CNP:99]: This should be reveresed if final test-def version changes are merged to `master`.

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 4]
```diff
@@ -27,8 +27,10 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.auth.password}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575-java/
-    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-399-java/
+    # TODO: remove TEMPORARY OVERRIDE to test ElasticSearch, reinstate: DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575-java/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-1279-java/
```
Comment (mattnayler) [inline charts/ccd-data-store-api/values.preview.template.yaml:31]: This should be reveresed if final test-def version changes are merged to `master`.

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 5]
```diff
@@ -27,8 +27,10 @@ java:
...
+    # TODO: remove TEMPORARY OVERRIDE to test ElasticSearch, reinstate: USER_PROFILE_HOST: http://ccd-user-profile-api-pr-399-java/
+    USER_PROFILE_HOST: http://ccd-user-profile-api-pr-497-java/
```
Comment (mattnayler) [inline charts/ccd-data-store-api/values.preview.template.yaml:33]: This should be reveresed if final test-def version changes are merged to `master`.

File: build.gradle [hunk 6]
```diff
@@ -277,8 +277,8 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.3'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.2-Categories'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.5_HMAN-368'
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.6-HMAN-368'
```
Comment (mattnayler) [inline build.gradle:281]: These should be updated to match final version numbers if being merged to `master`.

**Comment by marianadpereira** [comment]
closing as this is not needed anymore.

**Pull request closed without merging**